bcos-sync/bcos-sync/BlockSync.cpp

```diff
@@ -180,6 +180,31 @@ void BlockSync::printSyncInfo()
                        << "            --------------------------------------------";
 }
 
+void BlockSync::printBehindPeers()
+{
+    static auto lastLogUtcTime = utcTime();
+    if (utcTime() - lastLogUtcTime <= m_blockNumLogInterval)
+    {
+        return;
+    }
+
+    auto knownHighestNumber = m_config->knownHighestNumber();
+    auto blockNumThreshold = m_blockNumThreshold;
+    m_syncStatus->foreachPeer([knownHighestNumber, blockNumThreshold](PeerStatus::Ptr _p) {
+        if (std::abs(_p->number() - knownHighestNumber) > blockNumThreshold)
```

Comment:
add if node is sealer, observer not print error log

Reply:
fixed